***
//내가 포기 할 때까진 아직 끝난 게 아니다
***
```ruby

package org.sesac.todo_04.exception;

public class TodoNotFoundException extends Exception {

	public TodoNotFoundException(String string) {
		super(string);
	}

}


```
